### PR TITLE
Added ODS-498 for text verification

### PR DESCRIPTION
**Pull request by tarukumar**
Signed-off-by: tarukumar <takumar@redhat.com>

### CODE REVIEW

File: tests/Resources/Page/OCPDashboard/InstalledOperators/InstalledOperators.robot [hunk 1]
```diff
@@ -62,4 +64,21 @@ Wait Until Uninstallation Completes
...
+    [Arguments]   ${operator}
+     Search Installed Operator          ${operator}
+     Sleep   1
+     Click Element       xpath=//a[@data-test-operator-row="${operator}"]
+     Wait until page contains           Description           timeout=10
+
+Check IF URL On The Page Is Commercial
+    [Arguments]  ${url}
+     FOR  ${value}  IN   @{verification_list}
+          Run keyword If       $value in $url     FAIL    URL doesn't look like commerial it conatins '${value}' in it
```
Comment (lugi0) [inline tests/Resources/Page/OCPDashboard/InstalledOperators/InstalledOperators.robot:83]: Typo in contains

**Review by lugi0 — APPROVED** [review]
Pointed out a typo, other than that looks good to me

File: tests/Resources/Page/OCPDashboard/InstalledOperators/InstalledOperators.robot (hunk 1 above)
Comment (tarukumar) [reply tests/Resources/Page/OCPDashboard/InstalledOperators/InstalledOperators.robot]: Done

**Comment by bdattoma** [comment]
@tarukumar the keyword "Get HTTP Status Code" is basically the same which is present in Dashboard.robot except for the Return statement. I wonder if it would be possible to bring the keyword in a common keyword file so to avoid the code duplication.

**Comment by tarukumar** [comment]
> 

@bdattoma Agree ...i think as of now we keep like this . once we start refactoring process, we can move these to common file.

**Comment by bdattoma** [comment]
maybe, line 27 ($href == "mailto:undefined") may need to add double quotes to the href variable. 

About the refactoring, okay for me because we should involve two TCs to change it.

**Comment by tarukumar** [comment]
> maybe, line 27 ($href == "mailto:undefined") may need to add double quotes to the href variable.
> 
> About the refactoring, okay for me because we should involve two TCs to change it.

Robot allows you to use the variable in the expression without quotes by omitting the braces

**Comment by tarukumar** [comment]
This is the Jenkins PR Job execution : [PR-176](https://opendatascience-jenkins-csb-rhods.apps.ocp-c1.prod.psi.redhat.com/job/rhods-ci-pr-test/176/console) 
There are 5 TC's which are failed nit related to operator. The TC added in this PR is successfully executed.

**Review by lugi0 — APPROVED** [review]

File: tests/Resources/Page/OCPDashboard/InstalledOperators/InstalledOperators.robot [hunk 2]
```diff
@@ -68,6 +71,24 @@ Operator Should Be Uninstalled
   [Arguments]  ${operator}
   Page Should Not Contain Element  //a[@data-test-operator-row="${operator}"]
 
+
+Switch To New Tab
+    [Arguments]  ${tabname}
+     Click Element        //a[contains(text(), "${tabname}")]
```
Comment (bdattoma) [inline tests/Resources/Page/OCPDashboard/InstalledOperators/InstalledOperators.robot:77]: can this be used as common keyword?

**Review by bdattoma — APPROVED** [review]

File: tests/Resources/Page/OCPDashboard/InstalledOperators/InstalledOperators.robot (hunk 2 above)
Comment (tarukumar) [reply tests/Resources/Page/OCPDashboard/InstalledOperators/InstalledOperators.robot:77]: Totally we could do that. May be in refactoring process we could do that.

**Pull request merged**